bin/runner.js

```diff
@@ -35,7 +35,11 @@ function generate(model, allTestsGenerated) {
     return;
   }
 
-  var app = Elm.VerifyExamples.worker(model);
+  var app = Elm.Elm.VerifyExamples.init({ flags: model });
+  app.ports.reportError.subscribe(function(err) {
+    console.error(err);
+    process.exit(-1);
```

Comment:
☁️ `process.exit(-1)` will return an exit code to the calling process of 255. Is that intentional? IME, it's more usual to use an exit code of 1 (non-negative) to mean a problem in program start-up – like bad command-line arguments – and exit 2 to mean a runtime error.

Reply:
yes, it should be 1